dace/codegen/targets/cuda.py

```diff
@@ -1285,12 +1297,12 @@ def get_kernel_dimensions(self, dfg_scope):
         #       kernel, raise an invalid SDFG exception and recommend
         #       overapproximation.
 
-        return grid_size, block_size, True
```

Comment:
Shouldn't there be a case here that if has_dtbmap and has_tbmap are both true you fail?